Library/Homebrew/formula_installer.rb

```diff
@@ -1326,4 +1338,12 @@ def forbidden_license_check
         #{SPDX.license_expression_to_string formula.license}.
     EOS
   end
+
+  sig { params(keg: Keg).void }
+  def dsymutil(keg)
+    keg.dsymutil
+  rescue RuntimeError => e
```

Comment:
This seems a bit broad.

Reply:
Removed as part of inlining in e46a61e181f8e324d1bc4222bedf692c11ad84fa